src/rust/cryptography-x509-verification/src/policy/extension.rs

```diff
@@ -435,19 +476,6 @@ mod ee {
             _ => (),
         };
 
```

Comment:
We should leave a comment mentioning that the subject check happens elsewhere. (See: `basic_constraints` for an example comment like this)